flink-runtime/pom.xml

```diff
@@ -210,7 +210,12 @@ under the License.
 			<version>${curator.version}</version>
 			<scope>test</scope>
 		</dependency>
-	</dependencies>
+        <dependency>
```

Comment:
Why do you add a dependency to itself (this is the pom file of `flink-runtime`)?